src/app/components/not-found/not-found.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-not-found',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
  selector: "app-not-found",
```